src/v/kafka/server/handlers/describe_producers.cc

```diff
@@ -34,11 +34,11 @@ namespace {
 
 using ret_res = checked<cluster::rm_stm::transaction_set, kafka::error_code>;
 
-int64_t
-convert_rm_stm_time_to_milliseconds(cluster::rm_stm::time_point_type time) {
-    auto diff = cluster::rm_stm::clock_type::now() - time;
-    auto now = std::chrono::steady_clock::now();
-    return now.time_since_epoch() / 1ms - diff / 1ms;
+static int64_t
+convert_rm_stm_time_to_ts_ms(cluster::rm_stm::time_point_type time) {
+    auto diff = (cluster::rm_stm::clock_type::now() - time) / 1ms;
+    auto now = std::chrono::system_clock::now().time_since_epoch() / 1ms;
```

Comment:
I think the preference is to use ss::lowres_system_clock() instead chrono::system_clock for performance.